### PR TITLE
Ensuring the content dir on launch

### PR DESCRIPTION
**Pull request by radujipa**
Related to https://github.com/KanoComputing/peldins/issues/2247

### CODE REVIEW

**Comment by radujipa** [comment]
@alex5imon @convolu could you review please?

File: bin/make-snake [hunk 1]
```diff
@@ -96,6 +96,17 @@ function colourEcho
     colour_echo "$1"
 }
 
+function ensureContentDir
+{
+    # ensure that the content dir exists and it contains the default theme
+    for f in ~/Snake-content/*.xml; do
```
Comment (convolu) [inline bin/make-snake]: What happens here if Snake-content doesn't exist? What does this expand to?

File: bin/make-snake [hunk 2]
```diff
@@ -96,6 +96,17 @@ function colourEcho
     colour_echo "$1"
 }
 
+function ensureContentDir
+{
```
Comment (convolu) [inline bin/make-snake]: you could add something like:
`mkdir -p ~/Snake-content` 
which I think it a no-op if the folder already exists here

File: bin/make-snake (hunk 1 above)
Comment (radujipa) [inline bin/make-snake]: if Snake-content isn't there, it still triggers and creates the dir + copies the file

File: bin/make-snake (hunk 2 above)
Comment (radujipa) [inline bin/make-snake]: Awesome spot! :+1: 

Without the `-p` it prints `mkdir: cannot create directory '/home/.../Snake-content': File exists`

File: bin/make-snake (hunk 1 above)
Comment (convolu) [inline bin/make-snake]: It does get executed once, but it is not really obvious :-)

Comment (radujipa) [inline bin/make-snake]: that's because it is bash :)

**Comment by Ealdwulf** [comment]
Looks good to me :+1:

**Comment by radujipa** [comment]
@Ealdwulf @convolu thank you for the reviews :+1:  :ship:ing

**Pull request merged**